cmd/validate.go

```diff
@@ -122,5 +153,79 @@ func (o *ValidateOptions) SetAndValidate() error {
 		o.basePath = dir
 	}
 
+	switch OutputFormat(o.outputFormat) {
+	case YAML:
+	case JSON:
+	case LOG:
+	default:
+		return fmt.Errorf("unsuported output format")
+	}
+
+	if o.outputFile != "" && o.outputFormat == "" {
+		logrus.Warnf("ignoring --output-file as not --output-format is not specified")
```

Comment:
but we can output to a file the `log` format or not?

Reply:
Good catch. 
As mentioned in the [description of the --output-file](https://github.com/ykakarap/zeitgeist/blob/formatted_output/cmd/validate.go#L110) flag, it is only used if the output format is `yaml` or `json`. 
If the output-format is `log` it will print the output to stdout (as before).